tests/de/homeassistant_HassToggle.yaml

```diff
@@ -0,0 +1,11 @@
+language: de
+tests:
+  - sentences:
+      - "schalte den Küchenschalter"
+      - "schalte den Küchenschalter um"
+      - "betätige den Küchenschalter"
+      - "bitte schalte den Küchenschalter um"
```

Comment:
Butte is a skip word and doesn’t need to be added in sentences

Reply:
ah ok, I thought to test it at least once 🙈 
Will remove it

Reply:
Oh snap, you're right, this is a test. I didn't notice that. You should keep it here 🙈